phpBB/styles/prosilver/theme/common.css

```diff
@@ -603,6 +603,32 @@ a.header-avatar span:after {
 	display: none !important;
 }
 
+/* Smiley Dropdown
+---------------------------------------- */
+.smiley-container {
+	display: inline;
+	cursor: pointer;
+}
+
+.smiley-wrap {
```

Comment:
so this still needs dropdown-contents apparently until the dropdowns can be re-factored